packages/hooks/src/useGetPhoneNumberOTP.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { ValidationConstants } from '@deriv-com/utils';
+import { useMutation } from '@deriv/api';
+import { VERIFICATION_SERVICES } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+
+type THandleError = {
+    code: string;
+    message: string;
+};
+
+/** A hook for requesting OTP which is sent on whatsapp or sms platforms */
+const useGetPhoneNumberOTP = () => {
```

Comment:
can we rename this hook as 'useRequestPhoneNumberOTP' since this hook is for handling phone OTP requests